app/models/miq_server/log_management.rb

```diff
@@ -80,7 +80,7 @@ def pg_log_patterns
     return [] unless pg_data
 
     pg_data = Pathname.new(pg_data)
-    [pg_data.join("*.conf"), pg_data.join("pg_log/*")]
+    [pg_data.join("*.conf"), pg_data.join("pg_log/*"), Pathname.new("/etc/manageiq/postgresql.conf.d/*")]
```

Comment:
Is `/etc/manageiq/postgresql.conf.d/` the same everywhere?  If so, I'm good with this change.  We might want to externalize this path as a contanst like we do with `pg_data_dir` but if this is the only caller of it, it's fine to be like this for now.

Reply:
yes, path is pre-defined and the same in different repos: https://github.com/ManageIQ/manageiq-appliance/tree/master/COPY/etc/manageiq/postgresql.conf.d ,

https://github.com/ManageIQ/manageiq-appliance/blob/72e0313e25395f6396f54ca9ddb755afcd62bdf5/TEMPLATE/var/opt/rh/rh-postgresql95/lib/pgsql/data/postgresql.conf#L615 ,

  https://github.com/ManageIQ/manageiq-gems-pending/blob/ea3adf45d9aa2eb569c3d379bd3e0d4a509f4098/spec/support/mixins/pg_environment_updater.rb#L38

It will be the only caller in `ManageIQ/manageiq`